webpack/assets/javascripts/foreman_dashboard.js

```diff
@@ -0,0 +1,140 @@
+import $ from 'jquery';
+import { doesDocumentHasFocus } from './react_app/common/document';
+import { notify } from './foreman_toast_notifications';
+import { activateTooltips } from './foreman_tools';
+import { translate as __ } from './react_app/common/I18n';
+
+$(document).on('ContentLoad', () => {
+  // TODO: can we use the layout state to detect the page we are in?
+  if (window.location.pathname === '/') {
```

Comment:
@amirfefer @glekner any suggestions?

Reply:
This also needs to take the mounted location into account.

Reply:
mounted as the RAILS_RELATIVE_URL_ROOT?

Reply:
Yes

Reply:
@amirfefer does it make sense reusing your observeStore from ? https://github.com/theforeman/foreman/pull/6502/files

Reply:
It sounds like that respects the RAILS_RELATIVE_URL_ROOT and thus solve the problem. I also like it because it's not so tied into URLs but rather specific pages.

Reply:
`activeMenu` is only for marking high level menus, so submenus such as `dashboard` or `settings` won't assign into it.
plus it uses the same mechanism of ` window.location.pathname` so i'm not sure it will solve alternative relative url root